File: backend/core/src/modules/plots/plots.controller.ts
import { Controller, UseGuards } from '@nestjs/common';
import { ProjectScopedCrudController } from '../../common/access/project-scoped-crud.controller';
import { PlotsService } from './plots.service';
import { Plot } from '../../entities';
import { JwtAuthGuard } from '../auth/jwt-auth.guard';

@UseGuards(JwtAuthGuard)
@Controller('plots')
export class PlotsController extends ProjectScopedCrudController<Plot> {
  constructor(service: PlotsService) {
    super(service);
  }
}

File: backend/core/src/main/java/store/pjcloud/cowrite/core/controller/PlotsController.java
package store.pjcloud.cowrite.core.controller;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import store.pjcloud.cowrite.core.common.ProjectScopedCrudController;
import store.pjcloud.cowrite.core.entity.Plot;
import store.pjcloud.cowrite.core.service.PlotsService;

@RestController
@RequestMapping("/plots")
public class PlotsController extends ProjectScopedCrudController<Plot> {
    public PlotsController(PlotsService service) {
        super(service);
    }
}
